src/maintainer/knowledge_base.rst

```diff
@@ -1118,6 +1119,37 @@ Note that this requires ``conda>=4.8``. If you want to support older conda versi
 the requirement should be changed from ``run`` to ``run_constrained``. Note that
 ``conda<4.8`` will ignore the condition if it's a ``run_constrained`` on ``__osx``.
 
+Compile for Apple's Metal framework
+-----------------------------------
+
+It is possible to compile against Apple's Metal framework. 
+However, you will likely need to use newer SDKs (11 and newer) for functionality.
+If in doubt, you can search the `phracker/MacOSX-SDKs <https://github.com/phracker/MacOSX-SDKs.git>`__ 
+repo for the exact symbols and their availability.
+
```

Comment:
Maybe pyobjc-framework-metal can be used as an example?
https://github.com/conda-forge/pyobjc-framework-metal-feedstock/blob/main/recipe/conda_build_config.yaml